.github/workflows/terraform-observe_scheduler.yaml

```diff
@@ -34,21 +34,27 @@ jobs:
   terraform-test-apply:
     needs: get-test-directories
     if: ${{ needs.get-test-directories.outputs.directories != '[]' && needs.get-test-directories.outputs.directories != '' }}
-    name: OPAL Validation
+    name: Test Terraform Module
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         directory: ${{ fromJson(needs.get-test-directories.outputs.directories) }}
     steps:
-      - name: Validate OPAL with TF apply/destroy
-        uses: observeinc/.github/.github/actions/terraform-observe/terraform-test-apply@main
-        env: 
+      - uses: hashicorp/setup-terraform@v2
+        with: 
+          terraform_version: 1.2.9
```

Comment:
Do we want to hardcode a version here?

Reply:
This is preserving the action's behavior. I've been cutting useless declarations (e.g., noop tests, steps here with `id`) but avoiding behavioral changes. Aside from step names I guess. So no I don't want a hardcoded version here, but yes it's intentional that this PR doesn't change that. You can incorporate this in as a second case to remove this hardcoding, in addition to `tf-account`.